### DISCUSSION TITLE
Releases Hub servers Down? HTTP Status code: 500 Reason: 500

### DISCUSSION THREAD

**Discussion by wezley98** [dipien/releases-hub-gradle-plugin#133] [General]
**Describe the bug**
Running any task returns a `HTTP Status code: 500 Reason: 500`

**To Reproduce**
1. Run any task requiring Release Hub Server eg `./gradlew upgradeDependencies`
**Expected behavior**
List of dependencies to upgrade is returned

**Comment by maxirosson** [comment]
Fixed. It was an issue on our servers. Thanks for reporting it

**Comment by rk-puvvada** [comment]
@maxirosson is this issue still exists? Bcoz, when I performed a test build in my project after adding the releases-hub plugin running tasks like, **validateDependencies** or **listDependenciesToUpgrade** showing the same 500 error. Can you check is there an issues with the server?
